cudamapper/src/overlapper.cpp

```diff
@@ -171,6 +196,24 @@ void Overlapper::post_process_overlaps(std::vector<Overlap>& overlaps)
         fused_overlap.num_residues_                  = num_residues;
         overlaps.push_back(fused_overlap);
     }
+
+    if (drop_fused_overlaps)
```

Comment:
Can we have some unit tests for the `drop_fused_overlaps` option?

Reply:
Addressed in 4b30fe7.